.github/workflows/format.yml

```diff
@@ -0,0 +1,24 @@
+name: Code Format Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Small suggestion here: I think it's almost always worth to add `workflow_dispatch` trigger to the workflows. It will allow for running the workflow on specified branch, even if PR is not created (not very likely, but possible scenario).

Reply:
cc46482caf18ea0f339f056517e561d0b6f70aa8